app/views/bulk_update/recommendations_uploads/cancel.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render PageTitle::View.new(text: "Confirm that you want to cancel your bulk updates") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(text: "Back", href: url_for(:back)) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-l">Confirm you want to cancel bulk recommending trainees</h1>
```

Comment:
cc @johndoates - in the prototype i noticed this header "Confirm you want to cancel bulk recommending trainees" is different from the page title "Confirm that you want to cancel your bulk updates". Are they supposed to match?